### ISSUE TITLE
Value format for capture filter should allow mac anf ip formatted strings

### ISSUE THREAD

**Issue by shramroy** [open-traffic-generator/models#141]
Now capture filter only accepts hex strings. But It will be better if one can give input for mac  like this 00:00:00:00:00:01 and for ip like this 0.0.01

**Comment by ajbalogh** [comment]
@ashutshkumr this has the potential to be a breaking change in existing scripts - is this ok?

**Comment by shramroy** [comment]
> @ashutshkumr this has the potential to be a breaking change in existing scripts - is this ok?

Yes it will have breaking changes. But we can modify and let others notify.
